components/http_foundation.rst

```diff
@@ -420,12 +420,12 @@ conditional value specified in the client Request, use the
 :method:`Symfony\\Component\\HttpFoundation\\Response::isNotModified`
 method::
 
-    if ($response->isNotModified($request)) {
+    if (!$response->isNotModified($request)) {
```

Comment:
hm, ok. The documentation is still pretty confusing as written, though, because it presents one case (request not modified), but doesn't actually suggest what to do if the response *is* modified, because the response *still* needs to be sent.

I'd argue that ```isNotModified``` is badly named; it doesn't just test if the content is not modified; if it is, it also causes structural changes in the response that, while documented, are not implied by the method name. But that's an issue for a different PR.

Trying to use this with BinaryFileResponse is what triggered my confusion. It's unclear what the proper code path is when using BinaryFileResponse. A BinaryFileResponse doesn't actually have content set (so setting it to null has no effect), and its ```sendContent```  seems to unconditionally send the file.

Is this actually a problem in BinaryFileResponse, or is there something I'm missing?